subsys/bluetooth/host/adv.c

```diff
@@ -1192,6 +1192,7 @@ static int le_ext_adv_param_set(struct bt_le_ext_adv *adv,
 
 	cp->sid = param->sid;
 
+	cp->props = sys_cpu_to_le16(cp->props);
```

Comment:
Two comments:
1) The same issue appears to be in `bt_le_per_adv_set_param`
2) I think it would be more logical if we, instead of storing the properties directly as CPU endianess in `cp->props` (like `cp->props |= BT_HCI_LE_ADV_PROP_TX_POWER;`), we either
A) Add a local variable, `uint16_t props`, and then store them in that and then do `cp->props = sys_cpu_to_le16(props)` to be most consistent with the other `cp` assignments, or find a way store the bits directly as little endian (can you do `cp->props |= sys_cpu_to_le16(BT_HCI_LE_ADV_PROP_TX_POWER);`?)

Reply:
Sure - I'll look into that

Reply:
Changed to use a local `props` variable and added fix to `bt_le_per_adv_set_param`